api/openapi.yaml

```diff
@@ -1675,7 +1687,6 @@ components:
 
     AssetScanTemplate:
       type: object
-      x-go-type-name: "AssetScanTemplate"
```

Comment:
This should not change anything, but it allows bumping openapi generator version to v.1.13.0